### ISSUE TITLE
session is not preserved after calling `in_browser_session`.

### ISSUE THREAD

**Issue by msambarino** [HashNuke/hound#108]
After `in_browser_session` is called, session is always changed to the default session, irregardless of which was the current session before calling `in_browser_session`.

For example, say i have 3 sessions, `A`, `B` and `C` where `A` is the default session. if, while on session `B` i call 

```elixir 
in_browser_session "C", fn -> nil end
``` 
then current session after the call will be `A` instead of `B`.

I think it would be best if current session was preserved, something along the lines of: 
```elixir
def in_browser_session(session_name, func) do
    before_session = current_session_id
    change_session_to(session_name)
    apply(func, [])
    change_to_session(before_session)
  end
```

im willing to work on this and make a PR.

**Comment by tuvistavie** [comment]
Hi @msambarino, and thank you for reporting!
This should indeed behave as you said.
It would be really nice if you could send a PR.
Could you also fix #107 in the same PR please?
I have no strong opinion about using a function or `do`
but to avoid making a breaking change let's keep
the function for now and change the docs to reflect this.
Thank you very much! :smile:

**Comment by msambarino** [comment]
You're very welcome! I'll make a PR with both fixes (and #109 as well if its approved).

p.s. Thank you for your awesome work, i'm really enjoying using **hound**.

**Comment by tuvistavie** [comment]
Thank you very much!
I'm very happy you're enjoying it, I'm looking forward to your PR!
